### ISSUE TITLE
After running pipeline, metric gitlab_ci_pipeline_last_run_status stucks in state running

### ISSUE THREAD

**Issue by notariuss** [mvisonneau/gitlab-ci-pipelines-exporter#102]
curl -s 127.0.0.1/metrics | grep running
gitlab_ci_pipeline_last_run_status{kind="branch",project="myproj",ref="develop",status="running",topics="",variables=""} 1
But now my pipeline have another state..
```
        serviceMonitor:
          enabled: true
        config:
          projects_refs_polling_interval_seconds: 15
          gitlab:
            url: https://gitlab.com
            token: asdasd
            health_url: https://gitlab.com/
          defaults:
            fetch_pipeline_job_metrics: false
            refs_regexp: "^master$|^develop$"
          projects:
            - name: myproj/develop
```

**Comment by mvisonneau** [comment]
👋 hey @notariuss, thanks for raising this issue. Which version are you getting this from? 🤔

**Comment by mvisonneau** [comment]
it should be fixed in [0.3.2](https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/releases/tag/0.3.2)!
